### PR TITLE
Update sources-countries.js

### PR DESCRIPTION
**Pull request by DanHakimi**
Added Instagram group sources, edited Pinterest and Reddit. I think that third party reddit apps should properly be grouped with Reddit, as a social network, rather than treated as distinct sources.

### CODE REVIEW

File: static/components/dashboard/sources-countries.js [hunk 1]
```diff
@@ -42,34 +47,37 @@ customElements.define(
                 icon: "bing.com"
             },
             "Reddit": {
-                match: ["reddit.com", ".reddit.com"],
+                match: ["reddit.com", ".reddit.com",
+                    "com.laurencedawson.reddit_sync", "com.laurencedawson.reddit_sync.pro",
+                    "com.andrewshu.android.reddit".
```
Comment (ihucos) [inline static/components/dashboard/sources-countries.js:52]: That should be a comma, not a dot.

File: static/components/dashboard/sources-countries.js [hunk 2]
```diff
@@ -42,34 +47,37 @@ customElements.define(
...
-                    "www.pinterest.ie",
-                    "www.pinterest.it",
-                    "www.pinterest.jp",
-                    "www.pinterest.net",
-                    "www.pinterest.nz",
-                    "www.pinterest.ph",
-                    "www.pinterest.pt",
-                    "www.pinterest.ru",
-                    "www.pinterest.se"],
+                    "pinterest.at",
```
Comment (ihucos) [inline static/components/dashboard/sources-countries.js:59]: I believe I was getting e. G. `www.pinterest.com` as referrer, not `pinterest.com`. Where you getting it without the www? I am sure there's a reason for the change.

**Review by ihucos — CHANGES_REQUESTED** [review]
Thanks you so much!! Counting in traffic from the apps sounds sensible to me. Just a few points.

File: static/components/dashboard/sources-countries.js (hunk 2 above)
Comment (DanHakimi) [reply static/components/dashboard/sources-countries.js:59]: I get pinterest.co.uk.

Comment (ihucos) [reply static/components/dashboard/sources-countries.js:59]: ha,, interesting. Could you keep both - for me it was with the www. Then we should be good to go!

File: static/components/dashboard/sources-countries.js (hunk 1 above)
Comment (ihucos) [inline static/components/dashboard/sources-countries.js:52]: Ha, apologies, I was certain I added such a comment before: Can you replace the dot with a comma.

**Review by ihucos — CHANGES_REQUESTED** [review]

File: static/components/dashboard/sources-countries.js (hunk 1 above)
Comment (DanHakimi) [reply static/components/dashboard/sources-countries.js]: Alright, I... think I did that now. I'm using Github's web UI which... isn't the best way to do this, but I don't have a functional CLI setup right now and I'm not that great at git in the first place.

Comment (ihucos) [reply static/components/dashboard/sources-countries.js]: No worries.

File: static/components/dashboard/sources-countries.js [hunk 3]
```diff
@@ -42,12 +47,37 @@ customElements.define(
...
+                    "pinterest.fr",
+                    "pinterest.ie",
+                    "pinterest.it",
+                    "pinterest.jp",
+                    "pinterest.net",
+                    "pinterest.nz",
+                    "pinterest.ph",
+                    "pinterest.pt",
+                    "pinterest.ru",
+                    "pinterest.se"
```
Comment (ihucos) [inline static/components/dashboard/sources-countries.js:80]: Missing comma

**Review by ihucos — CHANGES_REQUESTED** [review]

**Comment by ihucos** [comment]
It works! Thanks a lot for the contribution.

**Comment by ihucos** [comment]
It's deployed btw

**Pull request merged**